requirements.txt

```diff
@@ -1,3 +1,4 @@
 # Example requirement, can be anything that pip knows
 # install with `pip install -r requirements.txt`, and make sure that CI does the same
 torch >= 1.8.1
+hydra-core >= 1.1.1
```

Comment:
I think that's something to discuss on whether we make this a first-level dependency, or a second tier like triton for instance)

Reply:
I would vote for keeping this an optional dependency, and only require it if users  want the `factory`

Reply:
sg! removed this in #93